api/src/Entity/ContentNode/Storyboard.php

```diff
@@ -37,59 +35,65 @@
     normalizationContext: ['groups' => ['read']],
 )]
 #[ORM\Entity(repositoryClass: StoryboardRepository::class)]
-#[ORM\Table(name: 'content_node_storyboard')]
 class Storyboard extends ContentNode {
-    #[ApiProperty(readableLink: true, writableLink: false)]
-    #[Groups(['read'])]
-    #[ORM\OneToMany(targetEntity: StoryboardSection::class, mappedBy: 'storyboard', orphanRemoval: true, cascade: ['persist'])]
-    public Collection $sections;
+    public const JSON_SCHEMA = [
+        'type' => 'object',
+        'additionalProperties' => false,
+        'required' => ['sections'],
+        'properties' => [
+            'sections' => [
+                'anyOf' => [
+                    [
+                        'type' => 'object',
+                        'patternProperties' => [
+                            // uuid4 key
+                            '[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}' => [
```

Comment:
Da wir das vermutlich noch oft brauchen werden: Gibt es eine Möglichkeit, das als globale #$def irgendwo in den JSON Schema Checker einzubauen? Oder geht das nicht weils ein Key ist?

Reply:
#2948